TCGstorageAPI/tcgapi_test.py

```diff
@@ -535,6 +538,51 @@ def test_tperSign_cert_fail(self):
         
         self.sedmock.invoke.return_value = status, rv, kwrv = (0x01, None, None)
         self.assertFalse(self.sed.get_tperSign_cert())
+    
+    def test_tper_attestation_cert_success(self):
+
+        self.sedmock.invoke.return_value = status, rv, kwrv = (0, [self.sed.cert], {})
+        type(self.sedmock).SSC = mock.PropertyMock(return_value='Enterprise')
```

Comment:
@adithya-22 Question, do we just test the value for Enterprise for the mock but not Opal?

Reply:
I think I might have missed that. Will add another test case for Opal.